visible_to/BUILD

```diff
@@ -0,0 +1,344 @@
+# Package groups defined for use in kubernetes visibility rules.
```

Comment:
To give a concrete example: once we untangle, we will be able to make use of e.g. `//pkg/kublet:__subpackages__` type visibility rules instead of explicit lists of packages. This visibility rule allows dependencies from packages in  the  ./pkg/kubelet/... subtree. More info here:

https://bazel.build/versions/master/docs/be/common-definitions.html#common.visibility